be/src/olap/tablet_meta_manager.h

```diff
@@ -34,13 +38,16 @@ class TabletMetaManager {
     static OLAPStatus get_json_header(DataDir* store, TTabletId tablet_id,
             TSchemaHash schema_hash, std::string* json_header);
 
-    static OLAPStatus save(DataDir* store, TTabletId tablet_id, TSchemaHash schema_hash, const TabletMeta* tablet_meta);
-    static OLAPStatus save(DataDir* store, TTabletId tablet_id, TSchemaHash schema_hash, const std::string& meta_binary);
+    static OLAPStatus save(DataDir* store, TTabletId tablet_id, TSchemaHash schema_hash, 
+                           const TabletMeta* tablet_meta, string header_prefix = "tablet_meta_");
```

Comment:
the default argument is not recommended.
The same to the following functions

Reply:
Mr zhao said we could use default argument.